spec/inertia/rendering_spec.rb

```diff
@@ -64,7 +64,7 @@
 
   context 'partial rendering' do
     let (:page) {
-      InertiaRails::Renderer.new('TestComponent', controller, request, response, '', props: { sport: 'hockey'}).send(:page)
+      InertiaRails::Renderer.new('TestComponent', controller, request, response, '', props: { sport: 'hockey' }).send(:page)
```

Comment:
The sneaky part is here: `.page` is compared to `response.body`, which is the subject of the test file. This test happily passed because both `.page` and `response.body` were filtering out all the props, leaving a comparison between `{ props: {} }` and  `{ props: {} }`.

It failed once modified to check for the actual content we wish to see.